tex/report.tex

```diff
@@ -134,8 +134,11 @@ \section{Methodology}\label{sec:method}
 Python, Ruby, and Bash, which do exactly the following:
 \begin{itemize}
     \item Fetch open repositories from GitHub, which have \ff{java} language
-    tag, have reasonably big but not too big number of stars, and are
-    of certain minimum size;
+    tag, have reasonably big but not too big number of stars, have either MIT or Apache License,
+    and are of certain minimum size;
+    \item Filter out repositories those contain samples, instead real project,
```

Comment:
@h1alexbel this description is too vague, I believe. Maybe we can give a link to your Python repo here?

Reply:
@yegor256 fixed